src/configuration/vimrc.ts

```diff
@@ -0,0 +1,96 @@
+import * as _ from 'lodash';
+import * as fs from 'fs';
+import * as path from 'path';
+import { IConfiguration, IKeyRemapping, IVimrcKeyRemapping } from './iconfiguration';
+import { vimrcKeyRemappingBuilder } from './vimrcKeyRemappingBuilder';
+
+class VimrcImpl {
+  public load(configuration: IConfiguration) {
+    if (configuration.vimrcPath) {
+      configuration.vimrcPath = VimrcImpl.expandHome(configuration.vimrcPath);
+      if (!fs.existsSync(configuration.vimrcPath)) {
+        return;
+      }
+    } else {
+      configuration.vimrcPath = VimrcImpl.findDefaultVimrc();
+      if (!configuration.vimrcPath) {
+        return;
+      }
+    }
+
+    let vimrcContent = fs.readFileSync(configuration.vimrcPath, { encoding: 'utf8' });
+    let lines = vimrcContent.split(/\r?\n/);
+
+    for (const line of lines) {
+      VimrcImpl.tryKeyRemapping(configuration, line);
+    }
+  }
```

Comment:
Should we save the updated vscodevim configuration? Otherwise, we'll need to do this re-syncing every time vscode is launched. 

If i update my vimrc, I'll need to re-launch vscode? Is that what most users will expect? Changes to the vscodevim settings on the otherhand don't require a restart.

Reply:
We could re-evaluate the vimrc if the user saves it from VSCode. We could also have a "source" command to manually re-evaluate